code/modules/admin/admin_verbs.dm

```diff
@@ -552,7 +552,7 @@ var/list/admin_verbs_snpc = list(
 		return
 
 	var/turf/epicenter = mob.loc
-	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Custom Bomb")
+	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Custom Bomb", "Cancel")
 	var/choice = input("What size explosion would you like to produce?") in choices

```

Comment:
Not needed

`var/choice = input("What size explosion would you like to produce?") as null|anything in choices`